proctor-codegen/src/main/resources/com/indeed/proctor/consumer/ant/groups.ftl

```diff
@@ -23,6 +26,8 @@ public class ${mainClassName} extends AbstractGroups {
 
     public static final ${mainClassName} EMPTY = new ${mainClassName}(ProctorResult.EMPTY);
 
+    public static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
nitpick: maybe it's safer to start from more strict access modifier unless there's a reason to make it public, so I suggest
```
private static final ObjectMapper MAPPER = new ObjectMapper();
```

Reply:
Fixed